facet-core/src/lib.rs

```diff
@@ -47,6 +47,9 @@ mod impls_ulid;
 #[cfg(feature = "time")]
 mod impls_time;
 
+#[cfg(feature = "jiff")]
```

Comment:
I'd strongly urge you to name this `jiff02`. `jiff 1.0` is going to be released this summer.

(Unless y'all are doing frequent breaking change releases and don't mind doing one to bump the Jiff version.)